src/mito2/src/worker.rs

```diff
@@ -593,35 +594,55 @@ pub(crate) struct StalledRequests {
     ///
     /// Key: RegionId
     /// Value: (estimated size, stalled requests)
-    pub(crate) requests: HashMap<RegionId, (usize, Vec<SenderWriteRequest>)>,
+    pub(crate) requests:
+        HashMap<RegionId, (usize, Vec<SenderWriteRequest>, Vec<SenderBulkRequest>)>,
```

Comment:
We should also update the `stalled_count()` method. Well, maybe we can reduce some potential issues if we can merge the two requests into one.

Reply:
fixed in 065cf78f1cea9ae9f502084c4c498ddadb3d3a94